<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>finance_page</class>
 <widget class="QMainWindow" name="finance_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1198</width>
    <height>593</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Finances Page</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>811</width>
      <height>521</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="financePage">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <widget class="QTableView" name="financeTable">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>40</y>
        <width>601</width>
        <height>431</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="searchLabel">
      <property name="geometry">
       <rect>
        <x>218</x>
        <y>18</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Search:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="searchLine">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>15</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Enter an item or date...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="financeSearchButton">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>15</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dateLine">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="dateLabel">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Date of Purchase</string>
      </property>
     </widget>
     <widget class="QLabel" name="itemLabel">
      <property name="geometry">
       <rect>
        <x>85</x>
        <y>90</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Item Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="itemLine">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="amountLabel">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>140</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Amount Spent</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="amountLine">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>160</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="noteLabel">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>190</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Notes</string>
      </property>
     </widget>
     <widget class="QPushButton" name="submitButton">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>290</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="noteLine">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>210</y>
        <width>113</width>
        <height>70</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="placeholderText">
       <string>Enter note here...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="editButton">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>320</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteButton">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>350</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="paycheckPage">
     <widget class="QTableView" name="paycheckTable">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>40</y>
        <width>601</width>
        <height>431</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="paycheckSearchLine">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>15</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Enter a date...</string>
      </property>
     </widget>
     <widget class="QLabel" name="paycheckSearchLabel">
      <property name="geometry">
       <rect>
        <x>218</x>
        <y>18</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Search:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="paycheckSearchButton">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>15</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="pdateLine">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="pdateLabel">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Date</string>
      </property>
     </widget>
     <widget class="QLabel" name="pamountLabel">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>120</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Paycheck Amount</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="pamountLine">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="pnoteLabel">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>250</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Note</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="noteEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>270</y>
        <width>121</width>
        <height>91</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Enter a note...</string>
      </property>
     </widget>
     <widget class="QLabel" name="spendLabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>190</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Spending %</string>
      </property>
     </widget>
     <widget class="QLabel" name="savingLabel">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>190</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Saving %</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="spendingPercent">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>210</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="savingPercent">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>210</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="submitPaycheck">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>370</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="editPaycheck">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>400</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deletePaycheck">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>430</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="earningsPage">
     <widget class="QTableView" name="earningsTable">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>40</y>
        <width>601</width>
        <height>431</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="edateLabel">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Date</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edateLine">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="espendLabel">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>110</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Spending</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="espendLine">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>130</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="esaveLabel">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>160</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Saving</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="esaveLine">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="esearchLabel">
      <property name="geometry">
       <rect>
        <x>218</x>
        <y>18</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Search:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="esearchLine">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>15</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="esearchButton">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>15</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QPushButton" name="esubmitButton">
      <property name="geometry">
       <rect>
        <x>55</x>
        <y>210</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="earningEditButton">
      <property name="geometry">
       <rect>
        <x>55</x>
        <y>240</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="edeleteButton">
      <property name="geometry">
       <rect>
        <x>55</x>
        <y>270</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="purchasesPageButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Purchases</string>
    </property>
   </widget>
   <widget class="QPushButton" name="paychecksPageButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Paychecks</string>
    </property>
   </widget>
   <widget class="QPushButton" name="accountSettings">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>490</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Account Settings</string>
    </property>
   </widget>
   <widget class="QLabel" name="currSpendingLabel">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>110</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Spending: $</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="spendingLine">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>110</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="currSavingLabel">
    <property name="geometry">
     <rect>
      <x>955</x>
      <y>140</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Savings: $</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="savingsLine">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>140</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="earningsPageButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Earnings</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1198</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>financeSearchButton</sender>
   <signal>clicked()</signal>
   <receiver>finance_page</receiver>
   <slot>searchPurchases()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>550</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchLine</sender>
   <signal>textChanged(QString)</signal>
   <receiver>finance_page</receiver>
   <slot>searchPurchases()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>447</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>financeTable</sender>
   <signal>viewportEntered()</signal>
   <receiver>finance_page</receiver>
   <slot>refreshPurchases()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>82</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>accountSettings</sender>
   <signal>clicked()</signal>
   <receiver>finance_page</receiver>
   <slot>accountSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>525</y>
    </hint>
    <hint type="destinationlabel">
     <x>93</x>
     <y>562</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>paychecksPageButton</sender>
   <signal>clicked()</signal>
   <receiver>finance_page</receiver>
   <slot>goToPaychecks()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>purchasesPageButton</sender>
   <signal>clicked()</signal>
   <receiver>finance_page</receiver>
   <slot>goToPurchases()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>15</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>submitButton</sender>
   <signal>clicked()</signal>
   <receiver>finance_page</receiver>
   <slot>addToPurchases()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>71</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editButton</sender>
   <signal>clicked()</signal>
   <receiver>finance_page</receiver>
   <slot>editPurchases()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>402</y>
    </hint>
    <hint type="destinationlabel">
     <x>79</x>
     <y>398</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteButton</sender>
   <signal>clicked()</signal>
   <receiver>finance_page</receiver>
   <slot>deletePurchases()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>448</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>financeTable</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>finance_page</receiver>
   <slot>contentClicked(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>647</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>606</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>paycheckSearchLine</sender>
   <signal>textChanged(QString)</signal>
   <receiver>finance_page</receiver>
   <slot>searchPaychecks()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>456</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>paycheckSearchButton</sender>
   <signal>clicked()</signal>
   <receiver>finance_page</receiver>
   <slot>searchPaychecks()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>604</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>544</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>paycheckTable</sender>
   <signal>viewportEntered()</signal>
   <receiver>finance_page</receiver>
   <slot>refreshPaychecks()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>403</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>72</x>
     <y>331</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spendingPercent</sender>
   <signal>valueChanged(double)</signal>
   <receiver>finance_page</receiver>
   <slot>spendPercent()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>savingPercent</sender>
   <signal>valueChanged(double)</signal>
   <receiver>finance_page</receiver>
   <slot>savePercent()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>submitPaycheck</sender>
   <signal>clicked()</signal>
   <receiver>finance_page</receiver>
   <slot>submitPaycheck()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>443</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>423</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editPaycheck</sender>
   <signal>clicked()</signal>
   <receiver>finance_page</receiver>
   <slot>editPaycheck()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>61</x>
     <y>475</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deletePaycheck</sender>
   <signal>clicked()</signal>
   <receiver>finance_page</receiver>
   <slot>deletePaycheck()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>503</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>492</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>paycheckTable</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>finance_page</receiver>
   <slot>paychecksClicked(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>515</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>380</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>earningsPageButton</sender>
   <signal>clicked()</signal>
   <receiver>finance_page</receiver>
   <slot>goToEarnings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>21</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>esearchLine</sender>
   <signal>textChanged(QString)</signal>
   <receiver>finance_page</receiver>
   <slot>searchEarnings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>492</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>esearchButton</sender>
   <signal>clicked()</signal>
   <receiver>finance_page</receiver>
   <slot>searchEarnings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>581</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>536</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>earningsTable</sender>
   <signal>viewportEntered()</signal>
   <receiver>finance_page</receiver>
   <slot>refreshEarningsPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>464</x>
     <y>356</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>esubmitButton</sender>
   <signal>clicked()</signal>
   <receiver>finance_page</receiver>
   <slot>submitEarning()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>42</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>earningEditButton</sender>
   <signal>clicked()</signal>
   <receiver>finance_page</receiver>
   <slot>editEarning()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>63</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edeleteButton</sender>
   <signal>clicked()</signal>
   <receiver>finance_page</receiver>
   <slot>deleteEarning()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>347</y>
    </hint>
    <hint type="destinationlabel">
     <x>58</x>
     <y>408</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>earningsTable</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>finance_page</receiver>
   <slot>earningsClicked(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>467</x>
     <y>428</y>
    </hint>
    <hint type="destinationlabel">
     <x>82</x>
     <y>459</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>searchPurchases()</slot>
  <slot>refreshPurchases()</slot>
  <slot>accountSettings()</slot>
  <slot>goToPaychecks()</slot>
  <slot>goToPurchases()</slot>
  <slot>addToPurchases()</slot>
  <slot>editPurchases()</slot>
  <slot>deletePurchases()</slot>
  <slot>contentClicked(QModelIndex)</slot>
  <slot>searchPaychecks()</slot>
  <slot>refreshPaychecks()</slot>
  <slot>spendPercent()</slot>
  <slot>savePercent()</slot>
  <slot>submitPaycheck()</slot>
  <slot>editPaycheck()</slot>
  <slot>deletePaycheck()</slot>
  <slot>paychecksClicked(QModelIndex)</slot>
  <slot>goToEarnings()</slot>
  <slot>searchEarnings()</slot>
  <slot>refreshEarningsPage()</slot>
  <slot>submitEarning()</slot>
  <slot>editEarning()</slot>
  <slot>deleteEarning()</slot>
  <slot>earningsClicked(QModelIndex)</slot>
 </slots>
</ui>
